cpp/include/cudf/functions.h

```diff
@@ -592,6 +592,149 @@ gdf_error gdf_extract_datetime_second(gdf_column *input, gdf_column *output);
 
 /* binary operators */
 
+/**
+ * @brief Binary operation function between gdf_scalar and gdf_column structs.
+ *
+ * The function performs the binary operation of a gdf_scalar operand and a
+ * gdf_column operand.
+ *
+ * The valid field in the gdf_column output will be 1 (by bit) when the two
+ * operands (vax and vay) are not null. Otherwise, it will be 0 (by bit).
+ *
+ * It is required to set in an appropriate manner the fields in the gdf_scalar and
+ * gdf_column structs due to that the binary operation will not be performed.
+ *
+ * @param out (gdf_column) Output of the operation.
+ * @param vax (gdf_scalar) First operand of the operation.
+ * @param vay (gdf_column) Second operand of the operation.
+ * @param ope (enum) The binary operator that is going to be used in the operation.
+ * @return    GDF_SUCCESS if the operation was successful, otherwise an appropriate
+ *            error code
+ */
+gdf_error gdf_binary_operation_v_s_v(gdf_column* out, gdf_scalar* vax, gdf_column* vay, gdf_binary_operator ope);
```

Comment:
The `vax` and `vay` parameter names are confusing. Can they be renamed to something more intuitive such as `lhs` and `rhs` or similar?